### DISCUSSION TITLE
크롬 스크린리더 확장프로그램 정보 공유💻

### DISCUSSION THREAD

**Discussion by chaewon0128** [marketkarly-team/marketkarly-client#56] [General]
[Screen Reader - 구글 확장프로그램](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn/related)

접근성 확인 작업시 사용해 보시면 좋을 거 같습니다 🙌

![image](https://user-images.githubusercontent.com/119389600/216212847-9381bfbe-cbd8-4872-abfb-83120c9bfe3f.png)
![image](https://user-images.githubusercontent.com/119389600/216212867-abe90b31-5b21-474b-9717-048f26901d33.png)
![image](https://user-images.githubusercontent.com/119389600/216212918-e5e0460b-e763-4e04-ada0-1a7af6452c0a.png)

**Comment by jaesukpark77** [comment]
저도 찾고 있었는데 정보 감사합니다!!!👍👍👍
